src/vivarium/framework/components/parser.py

```diff
@@ -252,10 +255,10 @@ def _clean_args(args: list, path: str) -> tuple:
                 raise ParsingError(f"Invalid component argument {a} for component {path}")
 
             out.append(a[1:-1])
-        return tuple(out)
+        return out
```

Comment:
could you explain why you changed this from a tuple to a list in the first place?

Reply:
I tried to reproduce the mypy error that led to this change and things ended up working without that change with a bit of typing actually so I've undone the list conversion and all related changes.